### DISCUSSION TITLE
Animations when switching between different widgets (example switching between frames)

### DISCUSSION THREAD

**Discussion by FarhanRah** [TomSchimansky/CustomTkinter#1313] [Ideas]
Would be great if animations were added into custom tkinter. For example, when the user switches between one frame to another, there could be an animation such as fade in or fade out.

**Comment by MaxNiark** [comment]
heeey 

Could you make  that with a copy of draw engine tkinter ? 

this possible, i had made it in the past when i was rework some widget display.

Now with our computer, Draw engine can work really good !!! Acctually, i get a code with a movement of frame :) 

Canvas can move easy peazy, the only way is to set the draw engine and get animation of color and display, like fade out with function. 

Perhaps, draw engine can't play in same time, so you need to wait the first execute widget display before the second lunch. ..
